Makefile

```diff
@@ -1,12 +1,19 @@
-IMAGE := quay.io/jparrill/ztp-pipeline
-TAG := latest
+CI_FOLDER = images
+
+PIPE_IMAGE = quay.io/jparrill/ztp-pipeline
```

Comment:
image from jparrill ? not from the org?

Reply:
Yep, I need to rebuild the image but we need this in main, so after the merge I will rebuild the image and update the things